torch/quantization/fake_quantize.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from torch.nn import Module
+from .observer import default_observer
+
+DTYPE_RANGE = {
```

Comment:
can we make `torch.iinfo(torch.qint8).min` work with quantized types? but I don't insist

Reply:
ok, will try, if it's easy we can do that, but it's not a priority

Reply:
done